lib/twilio/rtc/peerconnection.ts

```diff
@@ -916,7 +916,7 @@ PeerConnection.prototype.makeOutgoingCall = function(token, params, callsid, rtc
 
   function onOfferSuccess() {
     if (self.status !== 'closed') {
-      self.pstream.invite(self.version.getSDP(), self.callSid, self.options.preflight, params);
```

Comment:
Since we're not using options.preflight in this file, we should stop passing it from the call class.